src/App.css

```diff
@@ -0,0 +1,312 @@
+.add-title {
+  color: #888;
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+}
+
+.Line {
+  width: 97.5%;
+  height: 0.01rem;
+  margin: 2.5rem 0.063rem 1.813rem 0;
+  border: solid 1px #e8e8e8;
+}
+
+.book-individual {
+  width: 90%;
+  padding-top: 10px;
+  height: 10.625rem;
+  margin: 20px 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  list-style-type: none;
+}
+
+.books-container {
+  padding-top: 0.3rem;
+}
+
+.category-change {
+  padding: 0 20px;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.title-change {
+  font-family: 'Roboto Slab', sans-serif;
```

Comment:
- [ ] Kindly refer to the first comment for this CSS declaration for the `Roboto` font 👍